### PR TITLE
EPS-836: Twitter not showing correctly

### PR DESCRIPTION
**Pull request by vijetaR**
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by Bhikule19 — APPROVED** [review]

File: header-footer-elementor.php [hunk 1]
```diff
@@ -85,7 +85,12 @@ function hfe_enqueue_font_awesome() {
 			'5.15.3'
 		);
 
-		
+		wp_enqueue_style(
+			'hfe-widget-blockquote',
+			plugins_url( '/elementor-pro/assets/css/widget-blockquote.min.css', 'elementor' ),
```
Comment (sushmak02) [inline header-footer-elementor.php:90]: This path is from Elementor Pro, and we are not checking any condition of pro is installed

Comment (vijetaR) [reply header-footer-elementor.php:90]: Updated

Comment (sushmak02) [reply header-footer-elementor.php:90]: Adding && is not right here


Other styles will load only when both Elementor and pro plugins are active

Comment (vijetaR) [reply header-footer-elementor.php:91]: Updated to load them separately

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**